fsadmin/scripts/install-packages

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# This script installs additional packages used by the dependency image but
+# not needed by the runtime image, such as additional packages required to
+# build Python dependencies or install the Python package.
+
+# Bash "strict mode", to help catch problems and bugs in the shell
+# script. Every bash script you write should include this. See
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/ for details.
+set -euo pipefail
+
+# Display each command as it's run.
+set -x
+
+# Tell apt-get we're never going to be able to give manual feedback.
+export DEBIAN_FRONTEND=noninteractive
+
+# Update the package listing, so we know what packages exist.
+apt-get -y update
+
+# Get updates from base image
+apt-get -y upgrade
+
+apt-get -y install \
+    screen \
+    tmux \
+    git \
+    rsync \
+    emacs-nox \
+    vim \
+    psmisc \
+    sudo \
+    quota
+
+# Make life better. Come on, it's 2025.
+chsh -s /bin/bash root
```

Comment:
Nit: This isn't installing packages. Maybe move it to a `RUN` command directly in the Dockerfile?